### PR TITLE
fix the docker setup

### PR DESCRIPTION
**Pull request by antoinereyt**
# CHANGES:

The `.env` file is now optional.

If it not exists, environment variables will be used.

This allows you to use `.env` in local to ease development. 

And for deploy, where you do not want to put credentials in the image, you can use environment variables.

This will ease a lot the deploy on Cloud environments.

### CODE REVIEW

**Review by jalric — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -6,3 +6,5 @@ services:
       - .:/app
     ports:
       - "4001:4001"
+    env_file:
+      - .env
```
Comment (MrYawe) [inline docker-compose.yml]: Missing EOL 😄

**Review by MrYawe — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (antoinereyt) [reply docker-compose.yml]: fixed ;-)

**Review by MrYawe — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
-RUN curl https://install.zenaton.com | sh
+EXPOSE 4001
+
+COPY --from=composer /usr/bin/composer /usr/bin/composer
 
-# Install dependencies
+COPY ./bin /app/bin
+COPY ./src /app/src
+COPY ./start_zenaton /app/start_zenaton
 ADD composer.json composer.lock ./
```
Comment (pylebecq) [inline Dockerfile]: I think we can take the opportunity to also change this last `ADD` to a `COPY`. Should work exactly the same.

**Review by pylebecq — APPROVED** [review]
👏

**Pull request merged**